src/app/6-injectable/my-injectable2/my-injectable2.component.html

```diff
@@ -0,0 +1 @@
+{{this.myComponentVar}}
```

Comment:
Te pedí en un comentario que no pusieras this en el HTML, favor de revisar cada comentario en todo tu código de todas tus PRs antes de volver a poner los ejercicios como Re-check